packages/ui/src/theme/css/component/alert.scss

```diff
@@ -1,29 +1,29 @@
 .amplify-alert {
   align-items: var(--amplify-components-alert-align-items);
-  justify-content: var(--amplify-components-alert-justify-content);
   background-color: var(--amplify-components-alert-background-color);
+  justify-content: var(--amplify-components-alert-justify-content);
   padding-block: var(--amplify-components-alert-padding-block);
   padding-inline: var(--amplify-components-alert-padding-inline);
+}
+
+.amplify-alert--variation-info {
```

Comment:
We can use Sass stuff here to make this easier. Nest this under `.amplify-alert` and this could be `&--variation-info`